bazel/repository_locations.bzl

```diff
@@ -94,10 +94,10 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/gabime/spdlog/archive/v1.3.1.tar.gz"],
     ),
     com_github_google_libprotobuf_mutator = dict(
-        sha256 = "f45c3ad82376d891cd0bcaa7165e83efd90e0014b00aebf0cbaf07eb05a1d3f9",
-        strip_prefix = "libprotobuf-mutator-d1fe8a7d8ae18f3d454f055eba5213c291986f21",
-        # 2019-07-10
-        urls = ["https://github.com/google/libprotobuf-mutator/archive/d1fe8a7d8ae18f3d454f055eba5213c291986f21.tar.gz"],
+        sha256 = "54597f640c0ab5e5d783d2f3d3cfe8ad6da999ef1a194d89c2c5ab89a1fd8e13",
+        strip_prefix = "libprotobuf-mutator-dd89da92b59b1714bab6e2a135093948a1cf1c6a",
+        # 2019-10-08
```

Comment:
Why is this needed?

Reply:
I'll need to investigate this, but running with the fuzzer in the older version gives me this asan error (which is showing up on OSS-Fuzz)

```
evaluator_fuzz_test_with_libfuzzer: external/com_github_google_libprotobuf_mutator/src/mutator.cc:487: void protobuf_mutator::Mutator::Mutate(google::protobuf::Message *, size_t): Assertion `!keep_initialized_ || message->IsInitialized()' failed.
AddressSanitizer:DEADLYSIGNAL
=================================================================
==140620==ERROR: AddressSanitizer: ABRT on unknown address 0x6fad50002254c (pc 0x7f9cb7bdccfb bp 0x7f9cb7d29bc0 sp 0x7ffe9b1c9fd0 T0)
    #0 0x7f9cb7bdccfa in raise (/lib/x86_64-linux-gnu/libc.so.6+0x36cfa)
    #1 0x7f9cb7bc78ac in abort (/lib/x86_64-linux-gnu/libc.so.6+0x218ac)
    #2 0x7f9cb7bc777e  (/lib/x86_64-linux-gnu/libc.so.6+0x2177e)
    #3 0x7f9cb7bd5541 in __assert_fail (/lib/x86_64-linux-gnu/libc.so.6+0x2f541)
    #4 0xfa98e3d in protobuf_mutator::Mutator::Mutate(google::protobuf::Message*, unsigned long) (/usr/local/google/home/asraa/.cache/bazel/_bazel_asraa/f23f95d44d486dfd95cb67d4d7eb464d/execroot/envoy/bazel-out/k8-fastbuild/bin/test/extensions/filters/common/expr/evaluator_fuzz_test_with_libfuzzer+0xfa98e3d)
```

Reply:
Yeah, I think it's fine to bump if it fixes.